zeppelin-web/src/app/notebook/paragraph/paragraph-control.html

```diff
@@ -13,7 +13,24 @@
 -->
 
 <div id="{{paragraph.id}}_control" class="control" ng-show="!asIframe">
-
+  <span ng-show="paragraph.runtimeInfos.jobUrl.values.length == 1 && paragraph.runtimeInfos.jobUrl.group == 'spark'">
+    <a href="{{paragraph.runtimeInfos.jobUrl.values[0]}}" target="_blank" style="text-decoration: none;">
+      <span class="fa fa-tasks"></span>
+      {{paragraph.runtimeInfos.jobUrl.label}}
+    </a>
+  </span>
+  <span class="dropdown" ng-show="paragraph.runtimeInfos.jobUrl.values.length > 1 && paragraph.runtimeInfos.jobUrl.group == 'spark'">
+    <span style="cursor:pointer;color:#3071A9" tooltip-placement="top" tooltip="View in Spark web UI"
```

Comment:
Shell we remove condition `paragraph.runtimeInfos.jobUrl.group == 'spark'` here, too?
Also how about display tooltip from property of `runtimeInfos` instead of hardcoded "View in Spark web UI" ?